posts/2021-01-22-jwtbuilder-redirects-21001.adoc

```diff
@@ -0,0 +1,168 @@
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/14925[OAuth user registry lookups may use incorrect custom cache key]
++
+In previous releases, a flaw existed where a previously authenticated user’s `Subject` might not be found in the authentication cache during an OAuth authentication flow, when using a custom user registry. The cache key used to retrieve user claims from the authentication cache was based on the realm and username, but the correct cache key might be a combination of the OAuth provider name and the OAuth token object itself. The behavior has been corrected and the appropriate cache key should now be used.
++
+For more information visit the link:https://openliberty.io/docs/latest/reference/feature/oauth-2.0.html[Open Liberty Documentation].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/15220[Add HTTP/2 IOException for misbehaving client error case]
++
+It was previously possible for the HTTP/2 channel to throw a `NullPointerException` when it attempted to write out HTTP headers on a connection that had been terminated due to a connection error. Beginning in Open Liberty 21.0.0.1, the HTTP/2 channel will now throw a more informative `IOException` for this scenario. Read more about link:https://openliberty.io/blog/2018/06/29/full_java_ee_8_liberty_18002.html#http2[Open Liberty's support for HTTP/2].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/15342[CONTAINER_NAME env variable is not reflected in logstashCollector-1.0]
++
+Starting from Open Liberty 20.0.0.9, JSON logs created by the `logstashCollector-1.0` feature do not properly reflect the value set for the environment variable `CONTAINER_NAME`. The value set for `CONTAINER_NAME` should be reflected in the `serverName` field of the JSON logs, but the default server name from `wlp.server.name` is shown instead. This behaviour has been corrected for Open Liberty 21.0.0.1, visit the link:https://openliberty.io/docs/latest/forwarding-logs-logstash.html[Logstash Collector Documentation] for more information.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/15242[Stop the ACME Certificate Checker Task when the server is stopping]
++
+Support for the Automatic Certificate Management Environment (ACME) protocol was added in Open Liberty link:https://openliberty.io/blog/2020/09/25/signed-certificate-with-acme-200010.html#acme[20.0.0.10], enabling automatic fetching of browser-trusted TLS certificates from an ACME certificate authority. This release resolves a bug where the background task scheduled to check for expiring or revoked certification remains scheduled after the server enters quiesce phase. The task is now cancelled when the server is stopping. Read more about link:https://openliberty.io/docs/latest/acme-cert-management.html[Open Liberty's support for the ACME protocol].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/14936[Enable MyFaces 2.3.7 for Open Liberty]
```

Comment:
Wrong Link, should point at
https://github.com/OpenLiberty/open-liberty/issues/14950

Reply:
Fixed and committed.